src/applications/mhv-medical-records/components/shared/NewRecordsIndicator.jsx

```diff
@@ -99,6 +107,29 @@ const NewRecordsIndicator = ({
 
   const content = () => {
     if (refreshedOnThisPage) {
+      if (refreshPhase === refreshPhases.CALL_FAILED) {
+        return (
+          <va-alert
+            status="warning"
+            visible
+            aria-live="polite"
+            data-testid="new-records-refreshed-call_failed"
+          >
+            <h2>Your records may not be up to date.</h2>
+            <p>
+              There’s a problem with our system, and we can’t access the date
+              your records were last updated. We’re sorry.
+            </p>
+
+            <p> Please check backlater for updates.</p>
+
+            <p>
+              If it still doesn’t work, call us at 877-327-0022 (TTY:711). We’re
```

Comment:
Missing the link for the phone number. There's another example in that page. It's something like:
```
<va-telephone contact={CONTACTS.MY_HEALTHEVET} /> (
<va-telephone tty contact={CONTACTS['711']} />
```